src/components/SelectionList/ListItemRightElementWithArrow.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type ListItemRightElementWithArrowProps = {
+    enabled?: boolean;
+};
+
+function ListItemRightElementWithArrow({enabled}: ListItemRightElementWithArrowProps) {
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+
+    return (
+        <View style={styles.flexRow}>
+            <Text style={[styles.alignSelfCenter, styles.textSupporting, styles.pl2, styles.label]}>
```

Comment:
I am no engineer but just noting that not all of these table rows have the enabled/disabled text, right? Sometimes they might have nothing, or sometimes they might have a badge (Owner, Admin, card limit, etc). Not sure how that might change this abstraction.

Reply:
@shawnborton You're right that it can be a badge in other use cases. But this component will be only used by rows (of different components) to show the enabled status. 

Maybe rename it to `ListItemRightElementEnabledStatus` or `RightElementEnabledStatus` sounds better?

Reply:
Yeah so for this case, I have only abstracted the component used in the Categories, Tags, Taxes and Distance rates, The other usecases might have different styles or even different components inside as you mentioned. 

Abstracting this more might lead to loosing the benefit in this case as we might have to pass more props and styles which would again have to be defined individually in pages/ components this is used and it could lead to different styles.

I can rename this based on @eh2077 suggestion

Reply:
Cool, I don't feel too strongly really - mostly just noting that we have different row behaviors in various places. Thanks for hearing me out!

Reply:
I can certainly see us further abstracting this especially for the Badge style - Members with roles, Cards with limits etc. I think we can do that as we go though